Misc/NEWS.d/next/Library/2019-05-20-17-08-26.bpo-36972.3l3SGc.rst

```diff
@@ -0,0 +1 @@
+Add SupportsIndex protocol to allow type checking to detect classes that can have `hex()` `oct()` and `bin()` called on them.
```

Comment:
Comma is missing after `hex()`, also the formulation can be improved, you can just say `classes that can be passed to ...`. Finally, mention that it is added to typing module.